Add gscan2pdf (#258)
myfreeweb/awesome-gtk: README.md

### Document Scanners

- [Decoder](https://apps.gnome.org/Decoder) - QR Codes scanner and generator `#rust` `#gtk4` `#libadwaita` `#gnome`.
- [Document Scanner (SimpleScan)](https://apps.gnome.org/SimpleScan) - Document scanner using the SANE framework with crop and rotate editing features `#vala` `#gtk4``#libadwaita` `#gnome`.

> - [gscan2pdf](https://gscan2pdf.sourceforge.net/) - Document scanner that can scan, clean the scan and do OCR on the scan or imported images (incl. existing PDFs, DjVus or other file types), and make PDF and DjVu-files with embedded OCR-text `#perl` `#gtk3`.

Reviewer: ```suggestion
- [gscan2pdf](https://gscan2pdf.sourceforge.net) - Document scanner with cleaning and OCR features (on the scan or imported images incl. PDFs, DjVus or other file types) `#perl` `#gtk3`.
```

Reviewer: Tried to make the description more concise.
